src/Shortener.php

```diff
@@ -52,19 +68,20 @@ public function expand(string $shortUuid): string
         $number = BigInteger::zero();
         foreach (str_split(strrev($shortUuid)) as $char) {
             $number = $number
-                ->multipliedBy($this->dictionary->getLength())
-                ->plus($this->dictionary->getCharIndex($char));
+                ->multipliedBy($this->dictionary->length)
+                ->plus($this->dictionary->charIndexes[$char]);
```

Comment:
This syntax will lead to a notice when no character like this exists: that's an un-tested edge case.

Reply:
Fixed by ddb205b